sbomtool/go.work.sum

```diff
@@ -0,0 +1,3 @@
+github.com/containerd/typeurl v1.0.2 h1:Chlt8zIieDbzQFzXzAeBEF92KhExuE4p9p92/QmY7aY=
```

Comment:
Can you please document either in the commit message or in the PR description why you had to do this?